content/en/smart-contracts/fundamentals/aggregators

```diff
@@ -0,0 +1,81 @@
+
+The base interface requires 4 components:
+- Client who has data to upload
+- Aggregator platform (a type of storage solution) that clients can interact with to make a deal and retrieve proof of storage
+- Off-chain aggregation node to aggregate the subpiece data into a larger file
+- On-chain Aggregation smart contract that clients can interact with to request to make a deal and verify proof of storage
+
+### Proof of Deal Subpiece Inclusion (PoDSI)
+
+Proof of Deal Subpiece Inclusion (PoDSI) is motivated by a need for subpiece data uploads, to have verification and proof that the data was included in an associated deal on Filecoin. PoDSI is heavily used in the aggregated dealmaking workflow. 
+
+PoDSI is a proofing library that allows the generation of proof for each subpiece CID (within the large data segment) and stores it in an off-chain database. You can store 
+
+The proof consists of two inclusion proofs:
+- An inclusion proof of a sub-tree corresponding to the tree of the client’s data. Optionally, this sub-tree can be misaligned with the proving tree.
+- An inclusion proof of the double leaf data segment descriptor within the data segment index.
+
+See a sample of one of its implementations with [Lighthouse.storage](https://docs.lighthouse.storage/lighthouse-1/filecoin-virtual-machine/podsi-a-simple-overview).
+
+For further technical discussion, see the Filecoin Improvement Proposal (FIP) [here](https://github.com/filecoin-project/FIPs/discussions/512).
+
+### Off-chain aggregation and dealmaking
+
+1. Client submits subpiece data to an off-chain API. The API generates the subpiece metadata and informs an Aggregator (e.g. subpiece CID, known as pCID, and URL to download the CAR file). 
+
+2. The aggregator hosts an off-chain aggregation node, which downloads these subpiece CAR files and aggregates them into a larger CAR file. 
+
+3. Simultaneously, the aggregator aggregates indexed data segments (based on specs [here](https://github.com/filecoin-project/FIPs/discussions/512)). It runs the PoDSI proofing library and generates proofs for each subpiece pCID and stores it in an off-chain database. 
+
+4. The aggregator uses the [Boost API](https://boost.filecoin.io/experimental-features/fvm-contract-deals) (programmatic) or uses Lotus node (manual) to make storage deals with storage providers for the aggregated larger CAR file. 
+
+5. Storage Providers download the aggregated CAR file and publish storage deals. 
+
+6. Clients can query a proofing endpoint provided by the aggregator (example [here](https://docs.lighthouse.storage/lighthouse-1/filecoin-virtual-machine/podsi-and-deal-info), which will look up the subpiece CID (pCID) in the database and return the PoDSI proof, aggregated CID, and associated dealID.
+
+7. Clients can use the subpiece pCID for on-chain verification with the aggregation smart contract, which will verify merkle proof to ensure the pCID hash matches the commPa of the associated dealID. 
+
```

Comment:
Adding workflow diagram for off-chain aggregation

<img width="1525" alt="shapes 7" src="https://github.com/filecoin-project/filecoin-docs/assets/113331491/5821fef7-1a0e-4a32-87a9-54a7e2b888f0">